exchanges/exchange.go

```diff
@@ -600,7 +600,15 @@ func (b *Base) SetupDefaults(exch *config.Exchange) error {
 		log.Warnf(log.ExchangeSys, "%s orderbook verification has been bypassed via config.", b.Name)
 	}
 
+	if b.Accounts == nil {
```

Comment:
suggestion: I think we can just rm this check and flush this if set defaults is called again.